changelog/v1.13.0-beta26/expose_retry_backoff.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/6933
+    description: This feature exposes the base and max interval of the retry policy backoff settings
```

Should we mention how /where this is set?

---

Not sure what you mean here, could you give an example?

---

Basically why should customers care? 

```suggestion
    description: Expose the ability to set retry backoff policy's intervals. This can be set at the gateway, vs, or even the route level.
```

---

If it's okay with you I'll commit your change here if there's nothing more to add